packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -156,9 +156,10 @@ class ElevatedButton extends ButtonStyleButton {
   /// used to create a [WidgetStateProperty] [ButtonStyle.backgroundColor].
   ///
   /// Similarly, the [enabledMouseCursor] and [disabledMouseCursor]
-  /// parameters are used to construct [ButtonStyle.mouseCursor] and
+  /// parameters are used to construct [ButtonStyle.mouseCursor], and
   /// [iconColor], [disabledIconColor] are used to construct
-  /// [ButtonStyle.iconColor].
+  /// [ButtonStyle.iconColor], and [iconSize] is used to construct
+  /// [ButtonStyle.iconSize].
```

Comment:
Tiny nit: the triple-compound sentence is a little bit hard to follow.

I noticed that the other `ButtonStyleButton.allOrNull()` values aren't included in this documentation. Perhaps they're simple enough that no docs are necessary, or maybe they could all be included.

Reply:
`iconColor` docs are included in some of the docs, if not all values. I'll split the icon related sentences form cursor.